snooty.toml

```diff
@@ -24,3 +24,4 @@ php-library = "MongoDB PHP Library"
 
 [constants]
 php-library = "MongoDB PHP Library"
+driver-short = "PHP library"
```

Comment:
using "driver" so existing source constant references keep working. can rename all refs in a later PR if needed